test/tests/functional/pbs_qsub_two_args.py

```diff
@@ -93,3 +93,19 @@ def test_qsub_with_executable(self):
         cmd = 'qsub -- /bin/sleep 10'
         rv = self.du.run_cmd(self.server.hostname, cmd=cmd)
         self.assertEquals(rv['rc'], 0, 'qsub failed')
+
+    def test_qsub_with_option_executable(self):
+        """
+        submit a job with an option and executable
+        """
+        cmd = 'qsub -V -- /bin/sleep 10'
```

Comment:
Hirin once told me that it is better to pass run_cmd() a list of command arguments instead of a string.